src/posts/siparcs_2022/marshall_siparcs_2022.md

```diff
@@ -0,0 +1,105 @@
+
+A few months out, I’m grateful for the impactful experience I had this summer. I gained skills and experience writing more efficient code, incorporating version control into my work, and collaborating on GitHub. I also had the opportunity to begin writing my own software packages and learn how to access and work with cloud-hosted data. These all related to my main internship project which was to develop tutorials for working with remote sensing data using xarray with the goal of increasing the accessibility and user base of these datasets. Most of all, working with my mentors and getting to interact with other interns at NCAR made this experience feel more like fun than actual work and left me feeling motivated to continue developing my own skills and stay involved in the open-source software community.
+
+## What did I do?
+
+### Developed Jupyter Book tutorials
+
+Most of my work this summer focused on developing educational resources to help remote sensing data users incorporate Xarray into their workflows. This project was motivated by recent increases in the availability of cloud-based datasets and computational tools, and in preparation for increasing data volume from missions like [NISAR](https://nisar.jpl.nasa.gov/) and [SWOT](https://swot.jpl.nasa.gov/). These advances are exciting because they offer the potential to democratize scientific participation and reduce barriers to entry related to computational and storage resources ([Gentemann et al., 2021](https://doi.org/10.1029/2020AV000354)). Alongside these exciting developments is the need for accessible and informative educational resources that accompany data and tools.
+
+The Jupyter Book tutorials each focus on a different remote sensing dataset and contain Jupyter Notebooks detailing common steps of a research workflow. They demonstrate working with global ice velocity measurements derived from a combination of optical and synthetic aperture radar (SAR) satellite imagery as well as SAR radiometric terrain corrected (RTC) backscatter imagery. These data make use of cloud-native processing chains and cover a range of cloud-native and traditional data types and the tutorials include examples of reading directly from object storage as well as working with data locally. In choosing these examples, we focused on time series datasets that lend themselves to working with Xarray and provide valuable information about Earth’s surface processes that can be used in a range of climate-related applications such as glacier dynamics, hydrology, hazards, and studies of vegetative cover and extent.
+
```

Comment:
```suggestion
Here are the two books I created:
 1. [Using Xarray to examine cloud-based glacier surface velocity data](https://e-marshall.github.io/itslive/)

Based on the Inter-mission Time Series of Land Ice Velocity and Elevation ([ITS_LIVE](https://its-live.jpl.nasa.gov/)) dataset.

2. [Sentinel-1 RTC imagery workflows with Xarray](https://e-marshall.github.io/sentinel1_rtc/)

Using Sentinel-1 Radiometric Terrain Corrected (RTC) backscatter datasets processed and hosted by both [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc#overview) and [Alaska Satellite Facility](https://www.arcgis.com/home/item.html?id=3dd8d25559db4ba6aa0e1b6e8cb5d39a)
```

Not sure this will render correctly